packages/vkui/src/components/PanelHeaderButton/PanelHeaderButton.module.css

```
@@ -17,7 +17,7 @@
 .PanelHeaderButton--primitive {
   height: 48px;
   line-height: 48px;
-  padding: 0 12px;
+  padding: 0 10px;
```

```suggestion
  padding-inline: 10px;
```

можно сразу под `stylelint-logical-shorthands` поправить

---

а, нет, нельзя 😅  PR ведь ещё не смержен